cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,78 @@
 /// <reference types='cypress' />
 
+import { generateStudent } from '../support/generateStudent';
+
 describe('Student Registration page', () => {
+  const { firstName, lastName, email, mobile, address } = generateStudent();
+
   before(() => {
-    
+    cy.visit('/');
   });
 
-  it('', () => {
-    
+  it('should be possible to register a new student', () => {
+    cy.get('#firstName')
+      .type(firstName);
+
+    cy.get('#lastName')
+      .type(lastName);
+
+    cy.get('#userEmail')
+      .type(email);
+
+    cy.contains('label', 'Other')
+      .click();
+
+    cy.get('#userNumber')
+      .type(mobile);
+
+    cy.get('#dateOfBirthInput')
+      .type(`{selectall} 10.01.2001 {enter}`)
+
+    cy.get('.subjects-auto-complete__value-container')
```

Comment:
To select this element, you can also use `#subjectsInput`, this selector will be more specific than class